### ISSUE TITLE
[BUG] [Linter]

### ISSUE THREAD

**Issue by iamjameswalters** [Riverside-Healthcare/djLint#202]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: Ubuntu MATE 20.04
 - Python Version (``python --version``): 3.8.10
 - djLint Version (``djlint --version``): 0.7.6
 - template language: Handlebars

## Issue

Linter does not report missing closing tags for `{{#if}}`/`{{#each}}` statements (perhaps others?).

## How To Reproduce

Create a `file.hbs` containing:

```
{{#if asdf}}
    {{jkl}}
{{/if}}
```
Then, run ` djlint --profile=handlebars file.hbs`. This passes the linter (as it should). Remove the final line ( `{{/if}}` ), save and run again. This also passes the linter, which it should not.

**Comment by christopherpickering** [comment]
Thanks for adding this! - Currently the linter only checks for orphaned html tags, not template ones. I'll change this over to a feature request and we can work on adding it in. Thanks!
